src/Ses/SesClient.php

```diff
@@ -127,4 +130,20 @@ public static function generateSmtpPassword(CredentialsInterface $creds)
 
         return base64_encode($version . $signature);
     }
+
+    /**
+     * @internal
+     * @codeCoverageIgnore
+     */
+    public static function applyDocFilters(array $api, array $docs)
+    {
+        $b64 = '<div class="alert alert-info">This value will be base64 encoded on your behalf.</div>';
```

Comment:
Can you please update comment to say something like or something similar:

RawMessage is base64 encoded by the SDK internally.